tomviz/modules/ModuleSlice.cxx

```diff
@@ -219,20 +225,30 @@ void ModuleSlice::addToPanel(QWidget* panel)
   }
 
   QVBoxLayout* layout = new QVBoxLayout;
+  QFormLayout* formLayout = new QFormLayout;
```

Comment:
Consider using auto to avoid repetition of the class names. We should take a pass to make some of this more consistent.